### ISSUE TITLE
create `--inputDir` and `--outputDir` for `--transform`

### ISSUE THREAD

**Issue by petermr** [ContentMine/norma#39]
These arguments would allow transformation of subdirectories (e.g. `table/`, `image`) which contain multiple files of the same type.

**Comment by petermr** [comment]
added. Wait for reaction and bugs before closing

**Comment by tarrow** [comment]
I don't think you have committed your code. I couldn't see it either here or on your petermr fork.

I think I can see the use of this option but I'm not sure that adding yet another two options to our very long list is a good idea. For example having --inputDir is very similar to the CProject concept. Could the same effect not be achieved with file globbing?

Also, I'm hesitant (since I've not seen the code) as to what transformDirectory and transformSingleInput do. I think all the transforms should probably really be a "single input" and we should just run many of these to transform a whole directory.

I think having a table of what we can do and what the default input and output options are stored in an enum is probably a good idea but it does reduce the consistency considering all of the command line options are stored in args.xml.

**Comment by petermr** [comment]
Code compiles and doesn't test.
Tried to commit but problem with branches.
Happy to think about globbing - worried about inetraction with the shells (e.g. `*` metacharacters have to be escaped. Maybe this is OK
Note that `--inputDir` is actually subdirectory.
